pkgs/development/libraries/libqglviewer/default.nix

```diff
@@ -1,28 +1,38 @@
-{ lib, stdenv, fetchurl, qmake, qtbase, libGLU, AGL }:
+{ lib, stdenv, fetchurl, qmake, qtbase, libGLU }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libqglviewer";
   version = "2.9.1";
 
   src = fetchurl {
-    url = "http://www.libqglviewer.com/src/libQGLViewer-${version}.tar.gz";
```

Comment:
The tarball isn't available anymore just as the rest of the website (see the comment by @SuperSandro2000 below), either fetch from Github or remove the package

Reply:
```nix
  src = fetchFromGitHub {
    owner = "GillesDebunne";
    repo = "libQGLViewer";
    tag = "v${finalAttrs.version}";
    hash = "...";
  };
```

(make sure to recompute the hash and not copy over the one of the previous tarball)